docs/advanced/viewport-transition.md

```diff
@@ -45,6 +70,11 @@ See properties of [InteractiveMap](/docs/components/interactive-map.md).
 Remarks:
 - The props returned by the callback may contain transition properties. For example, during panning and rotating, the callback is invoked with `transitionDuration: 0`, meaning that the map movement instantly matches the change of the pointer. When panning or zooming with keyboard, the callback is invoked with a 300ms linear transition.
 - It is recommended that when programatically triggering a transition, always explicitly set the transition properties (interpolator, easing and duration).
+- "Set and forget": the values of the following props at the start of a transition carry through the entire duration of the transition:
```

Comment:
So in some sense these props should ideally only be supplied in the "render" that initiates a transition, but not in the following "render"?

Reply:
Correct.